### PR TITLE
util-linux: don't set _uuidd homedir

### PR DESCRIPTION
**Pull request by CameronNemo**

### CODE REVIEW

**Comment by maxice8** [comment]
What is wrong ?

File: srcpkgs/util-linux/template [hunk 1]
```diff
@@ -14,7 +14,6 @@ checksum=f261b9d73c35bfeeea04d26941ac47ee1df937bd3b0583e748217c1ea423658a
 
 # Create uuidd system account for uuidd.
 system_accounts="_uuidd"
-_uuidd_homedir="/run/uuidd"
```
Comment (maxice8) [inline srcpkgs/util-linux/template]: Maybe set to /var/empty ?

Comment (the-maldridge) [reply srcpkgs/util-linux/template]: This should be /var/empty.

Comment (CameronNemo) [reply srcpkgs/util-linux/template]: it is set to /var/empty as of a day ago, I forgot to ping maxice and resolve

**Pull request merged**